sdk/spring/azure-spring-boot/src/samples/java/com/azure/spring/aad/CustomWebServerSecurityConfig.java

```diff
@@ -0,0 +1,16 @@
+package com.azure.spring.aad;
+
+import com.azure.spring.aad.webapi.AADResourceServerWebSecurityConfigurerAdapter;
+import org.springframework.security.config.annotation.method.configuration.EnableGlobalMethodSecurity;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+
+@EnableWebSecurity
+@EnableGlobalMethodSecurity(prePostEnabled = true)
+public class CustomWebServerSecurityConfig extends AADResourceServerWebSecurityConfigurerAdapter {
+    @Override
+    protected void configure(HttpSecurity http) throws Exception {
+        super.configure(http);
+        // Do some custom configuration
```

Comment:
Can we use the code in sample project? `azure-spring-boot-sample-active-directory-webapp`.